plugins/modules/keycloak_authentication.py

```diff
@@ -79,6 +79,12 @@
                 description:
                     - Priority order of the execution.
                 type: int
+            subFlowType:
+                description:
+                    - For new subflows, optionally specify the type.
```

Comment:
If this is only used at creation, you should mention that, like:
```suggestion
                    - For new subflows, optionally specify the type.
                    - Is only used at creation.
```